catroid/src/main/java/org/catrobat/catroid/utils/StatusBarNotificationManager.java

```diff
@@ -79,7 +81,7 @@ public int createUploadNotification(Context context, String programName) {
 		Intent uploadIntent = new Intent(context, MainMenuActivity.class);
 		uploadIntent.setAction(Intent.ACTION_MAIN);
 		uploadIntent = uploadIntent.setFlags(Intent.FLAG_ACTIVITY_RESET_TASK_IF_NEEDED);
-		PendingIntent pendingIntent = PendingIntent.getActivity(context, notificationId, uploadIntent,
+		PendingIntent pendingIntent = PendingIntent.getActivity(context, getId(), uploadIntent,
```

Comment:
`getId() ≠ notificationId`, `notificationId = ATOMIC_INTEGER.get()`
Always be careful not to change behavior.